etc/inc/disable-programs.inc

```diff
@@ -1014,7 +1017,8 @@ blacklist ${HOME}/.local/share/wget
 blacklist ${HOME}/.local/share/wormux
 blacklist ${HOME}/.local/share/xplayer
 blacklist ${HOME}/.local/share/xreader
-blacklist ${HOME}/.local/share/zathura
+blacklist ${HOME}/.local/share/zathura*/
```

Comment:
```suggestion
blacklist ${HOME}/.local/share/zathura
```

Reply:
GitHub syntax highlighting made me break disable-programs.inc. Thanks to your keen eyes and report this is now fixed :+1: .

Reply:
No problem :)

When I saw it I was confused, as it looked like it could be an unrelated fix
for something else, but since nothing was said about zathura and since
zathura.profile wasn't changed to match it, I concluded that it was likely a
typo.